src/language/__tests__/schema-kitchen-sink.graphql

```diff
@@ -41,6 +41,8 @@ union AnnotatedUnionTwo @onUnion = | A | B
 
 scalar CustomScalar
 
+scalar StringEncodedCustomScalar = String
```

Comment:
@dschafer also suggested `:` IIRC. My preference for `=` was based on matching the rest of the SDL forms, but I agree that the implication of equality isn't correct.

I'm a bit worried that `:` is also overloaded - we most use it for defining the types or values to arguments but also aliases.

Maybe a keyword is best to make this as literate as possible. I kind of like `as`. Thoughts from others?